EIPS/eip-2384.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: 2384
+title: Difficulty Bomb Delay
+author: Eric Conner (@econoar)
+type: Standards Track
+category: Core
+status: Draft
+created: 2019-11-20
+---
+
+## Simple Summary
+The average block times are increasing due to the difficulty bomb (also known as the "_ice age_") and slowly accelerating. This EIP proposes to delay the difficulty bomb for another 4,000,000 blocks (~611 days).
+
+## Abstract
+Starting with `???_FORK_BLKNUM` the client will calculate the difficulty based on a fake block number suggesting the client that the difficulty bomb is adjusting around 9 million blocks later than previously specified with the Homestead fork.
+
+## Motivation
+The difficulty bomb has been activated and block times are on the rise from 13.1s to 14.4s. This will start to accelerate soon and we will see 20s block times in December 2019 and 30s block times around February 2020. This will slowly start to make the chain bloated and more costly to use. It's best to delay the difficulty bomb again to around the time of expected launch of the Eth2 finality gadget.
```

Comment:
Can you clarify this text with some dates or block numbers (e.g. it was activated 19th November with block XYZ). Also adding a link to the discussion on gitter in the references section would be nice.